package/crds/glue.aws.crossplane.io_crawlers.yaml

```diff
@@ -103,13 +331,14 @@ spec:
                   region:
                     description: Region is which region the Crawler will be created.
                     type: string
-                  roleArn:
+                  role:
```

Comment:
this is a breaking change // why changed roleArn to role ?

Reply:
I just see that I would also want to change the equivalent fieldname in the Job resource from "roleArn" to "role". (I did forget it there somehow)

In both cases (Crawler/Job) AWS Doc says Name and ARN of role are accepted and AWS API does so (which is the important part). So a fieldname "role" would make more sense than "roleName" or "roleArn".

However the two cases differ in what the AWS API returns as the content of the "role"-field.
- Job: here the AWS API gives as the content of the "role"-field in the same way we (or the user) provide it. If we provide the name of a role, we get just the name back. If we provide an ARN, we get an ARN back. That's nice -> easy isUpToDate-check ;)
- Crawler: here the AWS API is not so nice. We always get the role name back. (which is funny, because the AWS Doc hints we should only get the ARN back ...) In this case we need an extra check in isUpToDate to be able to provide the "Name or ARN"-functionality to the user. (see crawler/setup.go:203) 

In summary I would say as long as we can maintain that "Name or ARN"-functionality without any or too much extra complexity, we should do so and keep the AWS API given fieldname, which reflects that. 

How do you think about this?